tutor/src/screens/assignment-review/overview.js

```diff
@@ -404,14 +459,41 @@ const QuestionList = observer(({ ux, scores }) => {
       </StyledNoActivity>
     );
   }
-  return <HomeworkQuestions
-    questionsInfo={scores.questionsInfo}
-    questionType="teacher-review"
-    sectionLinkRenderer={(props) => <SectionInfo ux={ux} {...props} />}
-    headerContentRenderer={(props) => <QuestionHeader ux={ux} {...props} />}
-    questionInfoRenderer={(props) => <QuestionFreeResponse ux={ux} {...props} />}
-    footerContentRenderer={(props) => <QuestionFooter ux={ux} {...props} />}
-    styleVariant={ux.planScores.isReading ? 'reading' : 'submission'} />;
+
+  if(ux.planScores.isReading) {
+    return Object.keys(scores.groupQuestionsByPageTopic).map((key, index) => (
+      <div key={index}>
+        <StyledTopicHeader>
+          <h3>
+            <ArbitraryHtmlAndMath
+              html={key} />
+          </h3>
```

Comment:
The invision screens have a little bit of extra margin above the header, and I think we need it to visually separate the groups

<img width="319" alt="image" src="https://user-images.githubusercontent.com/34174/94585001-412daf00-0234-11eb-9bb2-970a0d0aeb50.png">